portfolio/README.md

```diff
@@ -1,5 +1,29 @@
 This directory is where Chiamaka will write all of her code!
 
 ```bash
+
+
 mvn package appengine:run
+
+
+TEST CODE
```

Comment:
Look into markdown syntax, you should clean up this file a bit.  For example, the commands can be put into code snippets and the all caps lines can be made into section titles

Reply:
ok